server/pkg/auth/auth.go

```diff
@@ -83,12 +83,17 @@ func CreateSessionToken() string {
 }
 
 // SetCookie creates an http cookie and sets it in the response
-func SetCookie(w http.ResponseWriter, name, token string) {
-	// TODO: Implement remember me functionality (MaxAge: 0)
+func SetCookie(w http.ResponseWriter, name, token string, rememberMe string) {
+	maxAge := SessionMaxAge
+
+	if rememberMe == "true" {
+		maxAge = 1000 * 365 * 24 * 60 * 60
```

Comment:
Can this be turned into a constant that's only one value -- maybe append a comment so we know exactly how long it is?